extensions/Cake.AzureBlobStorage.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.AzureBlobStorage
+NuGet: Cake.AzureBlobStorage
```

Comment:
This looks like a candidate to the blacklist given that there's another addin in Cake-Contrib providing the same feature https://github.com/cake-contrib/Cake.AzureStorage

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194